check_api/src/main/java/com/google/errorprone/dataflow/nullnesspropagation/inference/NullnessQualifierInference.java

```diff
@@ -100,8 +100,8 @@ public static InferredNullability getInferredNullability(Tree methodOrInitialize
         "Tree `%s` is not a lambda, initializer, or method.",
         methodOrInitializerOrLambda);
     try {
-      return inferenceCache.getUnchecked(methodOrInitializerOrLambda);
-    } catch (UncheckedExecutionException e) {
+      return inferenceCache.get(methodOrInitializerOrLambda);
+    } catch (CompletionException e) {
```

Comment:
```java
@throws CompletionException if a checked exception was thrown while loading the value
```
[CompletionFailure](https://javadoc.io/static/io.earcam.wrapped/jdk.compiler/1.8.132/com/sun/tools/javac/code/Symbol.CompletionFailure.html) is a runtime exception, so I think that you can catch it directly?